rotkehlchen/chain/evm/decoding/aura_finance/utils.py

```diff
@@ -93,6 +95,10 @@ def _query_aura_pools(evm_inquirer: 'EvmNodeInquirer') -> list[dict[str, Any]] |
             method_name='poolLength',
             contract_address=booster_contract.address,
         )
+        chunk_size, call_order = get_chunk_size_call_order(
+            evm_inquirer=evm_inquirer,
+            arbiscan_chunksize=ARBITRUM_ETHERSCAN_MULTICALL_CHUNK_SIZE,
+        )
```

Comment:
the chunksize depends on the number of arguments that the multicall receives. You gave `ARBITRUM_ETHERSCAN_MULTICALL_CHUNK_SIZE` because is what one of the cases used but I believe it can be increased in other cases? Have you checked that?

Reply:
for aura finance, this has to be 4. tried multiple values and only <=4 works.